PokeDemoTests/PokemonListPresenterTests.swift

```diff
@@ -0,0 +1,142 @@
+//
+//  PokeDemoTests.swift
+//  PokeDemoTests
+//
+//  Created by Rudr Bansal on 06/08/22.
+//
+
+import XCTest
+@testable import PokeDemo
+import Alamofire
+
+final class PokemonListPresenterTests: XCTestCase {
+    
+    var presenter: PokemonListPresenter!
+    
+    // MARK: - Lifecycle
+    
+    override func setUp() {
+        super.setUp()
+        presenter = PokemonListPresenter(service: Service.shared)
```

Comment:
You should not be injecting real Service object. Rather inject MockService object

Reply:
Covered in 95b810a9d180840d001ce309cd07301a9c2284de